client/views/root/canada_post.js

```diff
@@ -0,0 +1,3 @@
+import RootView from '../generic-settings-form';
```

Comment:
@allendav Good point. I don't think basing the RootView in the <div> we find would be a easy thing to implement... I will make that all the shipping services use a `shipping` RootView, so right now we would have `shipping`, `self-help`, and `shipping-label`. Is that ok?